src/components/PeopleTable.tsx

```diff
@@ -1,5 +1,42 @@
+import { useSearchParams } from 'react-router-dom';
+import { Person } from '../types';
+import { PersonLinks } from './PersonLinks';
+import { SearchLink } from './SearchLink';
+import classNames from 'classnames';
+
+type Props = {
+  people: Person[];
+};
+
 /* eslint-disable jsx-a11y/control-has-associated-label */
-export const PeopleTable = () => {
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const [searchParams] = useSearchParams();
+  const sort = searchParams.get('sort') || '';
+  const order = searchParams.get('order') || '';
+
+  const getSortIconClass = (field: string) => {
+    if (sort !== field || !sort) {
+      return 'fa-sort';
+    }
+
+    return order === 'desc' ? 'fa-sort-down' : 'fa-sort-up';
+  };
+
+  const sortParams = (value: string) => {
+    if (
+      searchParams.has('sort') &&
+      searchParams.has('order') &&
+      value === sort
+    ) {
+      return { sort: null, order: null };
```

Comment:
The `sortParams` function resets the sort and order to `null` when the current sort field is clicked again. This might not clear the sort state as expected because `useSearchParams` typically expects string values. Consider using empty strings or removing the parameters entirely to reset them.